fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/insert/InsertIntoTableCommand.java

```diff
@@ -146,13 +146,14 @@ private void runInternal(ConnectContext ctx, StmtExecutor executor) throws Excep
             Preconditions.checkArgument(plan.isPresent(), "insert into command must contain target table");
             PhysicalSink physicalSink = plan.get();
             DataSink sink = planner.getFragments().get(0).getSink();
-            String label = this.labelName.orElse(String.format("label_%x_%x", ctx.queryId().hi, ctx.queryId().lo));
 
             if (physicalSink instanceof PhysicalOlapTableSink) {
                 if (GroupCommitInserter.groupCommit(ctx, sink, physicalSink)) {
                     return;
                 }
                 OlapTable olapTable = (OlapTable) targetTableIf;
+                String label = this.labelName.orElse(
+                        ctx.isTxnModel() ? null : String.format("label_%x_%x", ctx.queryId().hi, ctx.queryId().lo));
```

Comment:
could u add some comment to explain why lable should be null when we in txn model

Reply:
done